### PR TITLE
Make 'mapping' variables 'const' in example files

### PR DESCRIPTION
**Pull request by AE7TB99**
This pull request addresses issue #17260 by making the `mapping` variables `const` in various example files.

### CODE REVIEW

**Review by masterleinad — APPROVED** [review]

**Review by bangerth — APPROVED** [review]
Awesome, thanks for contributing to the project -- I hope we're going to get many more patches from you!

Would you mind adding an entry to `doc/news/changes/minor` that explains what you did and that ensures that you're going to show up on the list of authors?

**Comment by AE7TB99** [comment]
I have added an entry to `doc/news/changes/minor` explaining the changes as requested. I hope everything is ok.

File: doc/news/changes/minor/20240725AE7TB99 [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Made `mapping` variables `const` in example files.
+<br>
+(AE7TB99, 2024/07/25)
```
Comment (masterleinad) [inline doc/news/changes/minor/20240725AE7TB99:3]:
```suggestion
( Lóránt Hadnagy, 2024/07/25)
```

Comment (AE7TB99) [reply doc/news/changes/minor/20240725AE7TB99:3]: There is an extra space at the front.

**Review by marcfehling — APPROVED** [review]

**Pull request merged**